core/ingest/pom.xml

```diff
@@ -101,6 +101,18 @@
 			<artifactId>spark-sql_2.11</artifactId>
 			<version>${spark.version}</version>
 		</dependency>
+		<dependency> <!-- Spark dependency -->
```

Comment:
did anything change that requires this new dependency?  I thought spark ingest was already in there and working.